python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -86,14 +90,12 @@ def preprocess_ondisk_dataset(
     if is_homogeneous:
         # Homogeneous graph.
         num_nodes = input_config["graph"]["nodes"][0]["num"]
-        edge_data = pd.read_csv(
-            os.path.join(
-                dataset_dir, input_config["graph"]["edges"][0]["path"]
-            ),
-            names=["src", "dst"],
+        edge_fmt = input_config["graph"]["edges"][0]["format"]
+        assert edge_fmt in ["numpy", "csv"], print(
```

Comment:
moving such sanity check into `read_edges` is more reasonable and so that code here is simpler.

Reply:
if more formats are added in the future, we just need to update `read_edges()`.